### ISSUE TITLE
prefer-to-have-count: warn about `.toHaveLength()`

### ISSUE THREAD

**Issue by EvgenyOrekhov** [playwright-community/eslint-plugin-playwright#284]
It's a feature request to add one more case to `prefer-to-have-count` rule.

The following code should be considered invalid:

```js
expect(await page.getByText("test").all()).toHaveLength(2);
```

The following code should be considered valid:

```js
await expect(page.getByText("test")).toHaveCount(2);
```
